plugins/pipelines/latestCommitEvent.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = schema.models.pipeline.base.extract('id');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/pipelines/{id}/latestCommitEvent',
+    options: {
+        description: 'Get latest commit event for a given pipeline',
+        notes: 'Return latest commit sha',
+        tags: ['api', 'pipelines'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'build', 'pipeline']
+        },
+
+        handler: async (request, h) => {
+            const { eventFactory } = request.server.app;
+
+            return eventFactory
+                .list({
```

Comment:
Do we need use `list` here? Using `find` with same parameter will get what you want.

Reply:
Thanks, a lot of lists are unnecessary.
Is the `find` method already there somewhere? I added `limit 1` because I couldn't find it.

Reply:
yeah `limit 1` should be sufficient